scripts/update_deps.py

```diff
@@ -689,6 +702,11 @@ def main():
 
     print('Starting builds in {d}'.format(d=abs_top_dir))
     for repo in repos:
+        # Only process the required Vulkan or VulkanSC Headers Repo
+        if repo.name.lower().find('vulkan') !=-1:
```

Comment:
Won't this logic break if we get a non-Vulkan named repo as a dependency?  Why not instead use a "--sc" flag above instead of API?